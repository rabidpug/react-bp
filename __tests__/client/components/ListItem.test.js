import ListItem from 'Components/ListItem';
import React from 'react';
import { shallow, } from 'enzyme';

describe(
  'ListItem', () => {
    it(
      'Should render the ListItem', () => {
        const wrapper = shallow( <ListItem /> );

        expect( wrapper ).toMatchSnapshot();
      }
    );
  }
);
