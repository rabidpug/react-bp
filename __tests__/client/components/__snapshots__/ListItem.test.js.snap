// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem Should render the ListItem 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ListItem />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "actions": Array [
        <IconText
          onClick={undefined}
          text="Mark Completed"
          type="check"
        />,
        <IconText
          onClick={undefined}
          text="Delete"
          type="close-square-o"
        />,
      ],
      "children": <Meta
        description={
          <div>
            Created On: Invalid Date at Invalid Date
          </div>
        }
        title={undefined}
      />,
      "extra": <Avatar
        icon="exclamation-circle-o"
        prefixCls="ant-avatar"
        shape="circle"
        size="default"
      />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "description": <div>
          Created On: Invalid Date at Invalid Date
        </div>,
        "title": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "actions": Array [
          <IconText
            onClick={undefined}
            text="Mark Completed"
            type="check"
          />,
          <IconText
            onClick={undefined}
            text="Delete"
            type="close-square-o"
          />,
        ],
        "children": <Meta
          description={
            <div>
              Created On: Invalid Date at Invalid Date
            </div>
          }
          title={undefined}
        />,
        "extra": <Avatar
          icon="exclamation-circle-o"
          prefixCls="ant-avatar"
          shape="circle"
          size="default"
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "description": <div>
            Created On: Invalid Date at Invalid Date
          </div>,
          "title": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
